CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+# Welcome to the `wdl` crates!
+
+Community contributions rock and we are psyched you're reading this document!
```

Comment:
Welcoming feedback on this opening.

I think it's important for a contributing guide to be friendly and welcoming to new folks; after all, we're trying to encourage them to join our effort. So I went pretty informal with the opening message here. Maybe there's a different tone we want to strike?

What do yall think?

Reply:
I'm personally fine with it; not sure what tone @claymcleod might want, though.